### DISCUSSION TITLE
I want to print the exact text as the file "basic_text_info.txt" to my customized tear down method. how can i do that?

### DISCUSSION THREAD

**Discussion by poojapatel14710** [seleniumbase/SeleniumBase#1604] [Q&A] [answered]
Hello sir,
Basically i want to use "basic_text_info.txt" logs to my customized teardown method because I want to add that in my customized report. so how can i get that details if my test case is failed?

The below logs is from archived_logs>basic_test_info.txt and i want to use that in my different file

```
demo/testCases/telegram_new_channel.py::TelegramNewChannelTest::test_02_back_icon
--------------------------------------------------------------------
Last Page: https://test.sociocs.com/channels/add
  Browser: chrome (107.0.5304.89)
   Driver: chromedriver (107.0.5304.62)
Timestamp: 1668079698  (Unix Timestamp)
     Date: Thursday, 10 November 2022
     Time: 4:58:18 PM  (India Standard Time, UTC+05:30)
--------------------------------------------------------------------
Traceback: File "C:\Automation\Python\lib\unittest\case.py", line 592, in run
    self._callTestMethod(testMethod)
  File "C:\Automation\Python\lib\unittest\case.py", line 550, in _callTestMethod
    method()
  File "C:\Automation\SeleniumBase\demo\testCases\telegram_new_channel.py", line 18, in test_02_back_icon
    wait.until(lambda driver: driver.current_url == "https://test.sociocs.com/channels/ad")
  File "C:\Automation\senium_base\lib\site-packages\selenium\webdriver\support\wait.py", line 90, in until
    raise TimeoutException(message, screen, stacktrace)
Exception: Message: 
```

**Comment by mdmintz** [comment, marked as answer]
If you call ``super().tearDown()`` from your customized ``tearDown()`` method, then the ``basic_text_info.txt`` file will be generated if the test has failed, and then you can use Python File-IO to get the data from the ``latest_logs`` folder.

```python
    def tearDown(self):
        # do things before regular tearDown()
        super().tearDown()
        # do things after regular tearDown()
```

This will get you the full folder path where the individual test logs are stored per test:

```python
import os

os.path.join(os.path.abspath("."), self.log_path, self._BaseCase__get_test_id())
```

Remember that ``basic_text_info.txt`` is only generated for failing tests.

**Reply by poojapatel14710** [reply]
Thank you so much, sir! The above solution worked and now I am able to attach all three files from the latest logs folder to the Allure report.

**Comment by poojapatel14710** [comment]
And i also want to add below part of  html report to allure report 
so, how can i do that?
![image](https://user-images.githubusercontent.com/76393417/201327777-23e70002-3ada-4977-a27d-2dfc1673a8f2.png)

**Reply by mdmintz** [reply]
``pytest-html`` is a separate plugin. I don't have direct access to that. You'll have to use old-fashioned Python File-IO parsing to extract the data that you need from your ``pytest-html`` report.

**Reply by mdmintz** [reply]
Some of that will be found in the ``basic_test_info.txt`` file, which will be easier to parse.

**Reply by poojapatel14710** [reply]
Okay, Thank you
